pajbot/modules/wide_emote_limit.py

```diff
@@ -0,0 +1,121 @@
+import logging
+
+from pajbot.managers.handler import HandlerManager
+from pajbot.modules import BaseModule, ModuleSetting
+
+log = logging.getLogger(__name__)
+
+
+class WideEmoteLimitModule(BaseModule):
+    ID = __name__.split(".")[-1]
+    NAME = "Wide Emote Limit"
+    DESCRIPTION = "Times out users who post too many wide 7TV or FFZ emotes"
+    CATEGORY = "Moderation"
+    SETTINGS = [
+        ModuleSetting(
+            key="max_emotes",
+            label="Maximum number of wide emotes that can be posted",
+            type="number",
+            required=True,
+            placeholder="",
+            default=15,
+            constraints={"min_value": 1, "max_value": 167},
+        ),
+        ModuleSetting(
+            key="max_width",
+            label="Maximum width of emotes in pixels. Emotes exceeding this width will be counted as wide",
```

Comment:
Key and label can be made clearer

The intent of the setting is to specify a width that makes the bot consider the emote wide
Should also make sure to make it clear if the user specified 128 whether a 128-width emote is wide or a 129-width emote is wide